src/faldbt/project.py

```diff
@@ -163,9 +163,9 @@ def get_sources(self) -> List[ParsedSourceDefinition]:
         return list(self.nativeManifest.sources.values())
 
 
-@dataclass(init=False)
 class DbtRunResult:
-    nativeRunResult: Optional[RunResultsArtifact]
```

Comment:
Are we losing autocomplete by removing these?

Reply:
Not in my editor, but this depends on which language server you're using. Should keep an eye on it.